radio/src/edgetx.cpp

```diff
@@ -419,6 +419,15 @@ void generalDefault()
   g_eeGeneral.hatsMode = HATSMODE_SWITCHABLE;
 #endif
 
+#if defined(DEFAULT_6POS_CALIB)
+  uint8_t defaultCalib[] = DEFAULT_6POS_CALIB;
+  StepsCalibData* calib = (StepsCalibData*)&g_eeGeneral.calib[DEFAULT_6POS_IDX];
+
+  for (int i = 0; i < 5; i++) {
```

Comment:
I forgot to ask... should this not have been `DEFAULT_6POS_IDX` rather than 5? ;) Magic numbers?

Reply:
No, it is 6 positions, so 5 limits points

Reply:
Hm... interesting... DEFAULT_6POS_IDX for TX16S is 5, and DEFAULT_6POS_IDX for Boxer is 6 :thinking:

Reply:
Yes, that the IDX in the analog channel list (aka analog channel number)

Reply:
Ah, ok, with you now... thanks :)